documentation/modules/rhv-prerequisites.adoc

```diff
@@ -8,11 +8,23 @@
 
 The following prerequisites apply to {rhv-full} migrations:
 
-* You must use a xref:compatibility-guidelines_{context}[compatible version] of {rhv-full}.
-* You must have the {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate.
+* To create a source provider, you must have at least the `UserRole` and `ReadOnlyAdmin` roles assigned to you. These are the minimum required permissions, however, any other administrator or superuser permissions will also work.
+
+[IMPORTANT]
+====
+You must keep the `UserRole` and `ReadOnlyAdmin` roles until the virtual machines of the source provider have been migrated. Otherwise, the migration will fail.
+====
+
+* To migrate virtual machines:
+** You must have one of the following:
+*** {rhv-short} admin permissions. These permissions allow you to migrate any virtual machine in the system.
+*** `DiskCreator` and `UserVmManager` permissions. These permissions have to be granted to you individually by an `admin` user per each individual virtual machine that you want to migrate.
```

Comment:
I think the second part of this sentence is too detailed - it would be ok if the specified user is part of a group that are granted with these permissions (so it's not granted individually for the user) and it's also ok to have permissions on the cluster that propagate to all the vms within the cluster (so it's not individually per VM)

how about replacing this with:
```suggestion
*** `DiskCreator` and `UserVmManager` permissions for the virtual machines you want to update.
```